node/pkg/driver/node.go

```diff
@@ -613,9 +619,81 @@ func (d *NodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 }
 
 func (d *NodeService) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
-	goid_info.SetAdditionalIDInfo(req.VolumeId)
+	volumeId := req.VolumeId
+	goid_info.SetAdditionalIDInfo(volumeId)
 	defer goid_info.DeleteAdditionalIDInfo()
-	return nil, status.Error(codes.Unimplemented, "NodeGetVolumeStats is not implemented yet")
```

Comment:
what about enabling the [community `NodeGetVolumeStats` tests](https://github.com/IBM/ibm-block-csi-driver/blob/275618b2b53acf9c72e57f5ad30a89153801d1a6/scripts/csi_test/common_csi_tests_to_skip#L5)?

Reply:
https://github.com/IBM/ibm-block-csi-driver/pull/480